cartographer_ros/configuration_files/assets_writer.lua

```diff
@@ -7,11 +7,11 @@ options = {
       -- action = "fixed_ratio_sampler",
       -- sampling_ratio = 1.,
     -- },
-    -- {
-      -- action = "min_max_range_filter",
-      -- min_range = 1.,
-      -- max_range = 150.,
-    -- },
+    {
```

Comment:
It is respected when extracting the laser scan. The hokuyo seem to return some ringing though which is filtered out.

As of having this enabled: I think it is unclear, but in any case for any lasers these are pretty sane values for reasonable looking assets. So I want to keep it for now.